libs/core-ui/.eslintrc.json

```diff
@@ -16,9 +16,7 @@
     },
     {
       "files": ["*.ts", "*.tsx"],
-      "rules": {
-        "max-lines": "error"
```

Comment:
why do we remove this rule for core-ui? I didn't see we set global max-lines rule in this pr

Reply:
It is in .eslintrc.custom.eslintrc as a warn.
I was planning to fix all in this pr, but seems it is too big already, I will fix all max line issues in following pr.